### ISSUE TITLE
guest-wait page is very plain

### ISSUE THREAD

**Issue by dkozel** [bigbluebutton/greenlight#2085]
We were looking for a way for several guest presenters to join a room ahead of time, setup, and then admit the rest of the participants. Turning on the option to require a moderator to admit participants seemed like that would work, but instead of the nice waiting page that is displayed before a room starts:
![Screenshot from 2020-09-14 00-20-32](https://user-images.githubusercontent.com/253888/93031104-2e09c500-f620-11ea-87a0-328f531b81bd.png)

Attendees are left seeing a blank page with "Please wait for a moderator to approve you joining the meeting."

It would be better if the same UI was used as when waiting for a room to start, but alter the loading text to indicate that the user needs to wait for a moderator to admit them.

**Comment by ffdixon** [comment]
Yes, we'll be doing some improvements to the guest waiting page in the upcoming BigBlueButton 2.3.
